tests/foreman/api/test_role.py

```diff
@@ -16,15 +15,13 @@
 )
 from nailgun import entities
 from requests.exceptions import HTTPError
-from robottelo import decorators
+from robottelo.decorators import bz_bug_is_open
 from robottelo.test import APITestCase
 
 
-@ddt.ddt
-class RoleTestCase(APITestCase):
-    """Tests for ``api/v2/roles``."""
-
-    @decorators.data(
+def data_generator():
+    """Returns data for role name"""
```

Comment:
Return a tuple containing data generators function to be used to generate role name?